devices/stm32f3x8.yaml

```diff
@@ -23,83 +23,6 @@ _copy:
   ADC3_4:
     from: ../svd/stm32f303.svd:ADC3_4
 
-GPIOC:
-  _add:
-    LCKR:
-      description: GPIO port configuration lock register
-      addressOffset: 0x1C
-      access: read-write
-      fields:
-        LCKK:
-          description: Lock Key
-          bitOffset: 16
-          bitWidth: 1
-        LCK15:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 15
-          bitWidth: 1
-        LCK14:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 14
-          bitWidth: 1
-        LCK13:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 13
-          bitWidth: 1
-        LCK12:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 12
-          bitWidth: 1
-        LCK11:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 11
-          bitWidth: 1
-        LCK10:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 10
-          bitWidth: 1
-        LCK9:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 9
-          bitWidth: 1
-        LCK8:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 8
-          bitWidth: 1
-        LCK7:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 7
-          bitWidth: 1
-        LCK6:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 6
-          bitWidth: 1
-        LCK5:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 5
-          bitWidth: 1
-        LCK4:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 4
-          bitWidth: 1
-        LCK3:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 3
-          bitWidth: 1
-        LCK2:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 2
-          bitWidth: 1
-        LCK1:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 1
-          bitWidth: 1
-        LCK0:
-          description: Port x lock bit y (y=0..15)
-          bitOffset: 0
-          bitWidth: 1
```

Comment:
I could remove these `LCKR` definitions because, thanks to the patch, `GPIOC` is now copied from `GPIOB`, which has this register already.

Reply:
Sadly this has also removed a very minor typo correction from the LCKR which is now called "Lok Key", but I'll fix that separately.